src/commands/ext-install.ts

```diff
@@ -42,11 +41,8 @@ marked.setOptions({
  */
 export const command = new Command("ext:install [extensionName]")
   .description(
-    "install an official extension if [extensionName] or [extensionName@version] is provided; " +
-      (experiments.isEnabled("extdev")
-        ? "install a local extension if [localPathOrUrl] or [url#root] is provided; install a published extension (not authored by Firebase) if [publisherId/extensionId] is provided "
-        : "") +
-      "or run with `-i` to see all available extensions."
+    "add a published extension to firebase.json if [publisherId/extensionId] is provided;" +
```

Comment:
Do we want to keep "published" naming here or use "uploaded extension" since it can be any extension in the registry?